spec/stoplight/notifier/bugsnag_spec.rb

```diff
@@ -0,0 +1,76 @@
+# coding: utf-8
+
+require 'spec_helper'
+require 'bugsnag'
+
+RSpec.describe Stoplight::Notifier::Bugsnag do
+  StoplightStatusChange = Stoplight::Notifier::Bugsnag::StoplightStatusChange
+
+  it 'is a class' do
+    expect(described_class).to be_a(Class)
+  end
+
+  it 'is a subclass of Base' do
+    expect(described_class).to be < Stoplight::Notifier::Base
+  end
+
+  describe '#formatter' do
+    it 'is initially the default' do
+      expect(described_class.new(nil, nil).formatter)
+        .to eql(Stoplight::Default::FORMATTER)
+    end
+
+    it 'reads the formatter' do
+      formatter = proc {}
+      expect(described_class.new(nil, formatter).formatter)
+        .to eql(formatter)
+    end
+  end
+
+  describe '#bugsnag' do
+    it 'reads the Bugsnag client' do
+      client = ::Bugsnag
+      expect(described_class.new(client, nil).bugsnag)
+        .to eql(client)
+    end
+  end
+
```

Comment:
This is missing some [tests for the options attribute](https://github.com/orgsync/stoplight/blob/d195eaadb37a085ff79f7bbd594010cf606b8feb/spec/stoplight/notifier/hip_chat_spec.rb#L36-L47).